### PR TITLE
Feature/list automations

### PR DESCRIPTION
**Pull request by guyts96**

### CODE REVIEW

**Review by souziel — APPROVED** [review]

File: client/src/components/UI/buttons/DropdownButton.svelte [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            >
+                {option}
+            </li>
+        {/each}
+    </ul>
+</div>
+
+<style>
+    .dropdown:hover .dropdown-menu {
+        display: block;
```
Comment (Plantin7) [inline client/src/components/UI/buttons/DropdownButton.svelte:43]: You can do like this with tailwind `@apply block`

File: client/src/components/others/SearchBar.svelte [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    export let debounce = 0;
+    export let id = `search${Math.random().toString(36)}`;
+
+    const dispatch = createEventDispatcher();
+
+    let prevValue = value;
+    let timeout: NodeJS.Timeout;
+    let calling = false;
+
+    function debounced(cb: () => void) {
```
Comment (Plantin7) [inline client/src/components/others/SearchBar.svelte:15]: Use the existing debounce function ? (To see) 🤔

File: client/src/services/automationService.ts [hunk 3]
```diff
@@ -26,4 +26,15 @@ export default class AutomationService {
...
+
+    static async triggerAutomation(id: string): Promise<void> {
+        const headers = new Headers();
+        headers.set('Content-Type', 'application/json');
+        return fetch(`${config.BASE_URL}/automation/${id}`, {
+            headers,
+            method: 'POST',
+        })
+            .then((res) => res.json())
+            .catch((err) => console.log(err));
```
Comment (Plantin7) [inline client/src/services/automationService.ts:38]: We will never go into the catch block.

**Review by Plantin7 — APPROVED** [review]
🦈

File: server/src/services/automationService.ts [hunk 4]
```diff
@@ -90,6 +95,31 @@ class AutomationService implements EventObserver {
         });
     }
 
+    async createAutomation(automation: Automation) {
+        const haAut: HaAutomation = {
+            id: automation.id,
+            alias: automation.name,
```
Comment (Lukyio) [inline server/src/services/automationService.ts:101]: 👍

**Review by Lukyio — APPROVED** [review]

File: server/src/services/automationService.ts (hunk 4 above)
Comment (Lukyio) [reply server/src/services/automationService.ts:101]: ♌

**Pull request merged**